### ISSUE TITLE
`fetch failed` when trying to enables indexes  - can't connect to localhost:7700 (solved: change localhost to 127.0.0.1)

### ISSUE THREAD

**Issue by joepio** [meilisearch/strapi-plugin-meilisearch#582]
After getting strapi + meilisearch fully working, I tried setting it up again from scratch, only to find `fetch failed` errors when I'm trying to toggle indexing for types.

<img width="1033" alt="Screenshot 2022-11-19 at 12 05 54" src="https://user-images.githubusercontent.com/2183313/202847602-19c5dea0-5fce-4710-900e-f5d2a34b4f0b.png">

In the strapi logs I see this:

```
[2022-11-19 11:59:26.853] error: meilisearch: fetch failed
[2022-11-19 11:59:26.853] http: POST /meilisearch/content-type (21 ms) 200
[2022-11-19 11:59:30.731] error: meilisearch: fetch failed
[2022-11-19 11:59:30.731] http: PUT /meilisearch/content-type (17 ms) 200
[2022-11-19 12:04:02.733] error: meilisearch: fetch failed
[2022-11-19 12:04:02.733] http: POST /meilisearch/content-type (18 ms) 200
```

- I tried with and without a master key. I can confirm from the meilisearch logs and the strapi UI that the master keys match and are succesfully loaded
- I can confirm that the address `http://localhost:7700` is working fine in the browser, and the strapi UI shows me that this address is being used.
- Stopping the meilisearch server doesn't seem to change the error
- **Changing the host to not use `localhost` helped**:

```
module.exports = ({ env }) => ({
  //...
  meilisearch: {
    config: {
      // Why is localhost not working?
      // host: "http://localhost:7700",
      host: "http://127.0.0.1:7700",
      // Your master key or private key
      apiKey: env('MEILI_MASTER_KEY'),
    }
  }
})

```

**Environment (please complete the following information):**
- OS: MacOS m1 Ventura 13.0.1 (22A400)
- Meilisearch version: 0.29.1
- strapi-plugin-meilisearch version: 0.8.0
- Browser: [e.g. Chrome version 90.0]
```
    "@strapi/plugin-i18n": "4.5.0",
    "@strapi/plugin-users-permissions": "4.5.0",
    "@strapi/strapi": "4.5.0",
    "better-sqlite3": "7.4.6",
    "strapi-plugin-meilisearch": "^0.8.0",
    "strapi-tiptap-editor": "^0.9.6"
```

**Comment by bidoubiwa** [comment]
Hey @joepio I tried it locally using localhost and it works fine. Is it possible that it comes from the `host` file on your machine?

**Comment by joepio** [comment]
@bidoubiwa thanks for reaching out!

I don't think so, my hosts file looks innocent:

```
##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting.  Do not change this entry.
##
127.0.0.1       localhost
255.255.255.255 broadcasthost
::1             localhost
# Added by Docker Desktop
# To allow the same kube context to work on the host and the container:
127.0.0.1 kubernetes.docker.internal
# End of section
```

**Comment by bidoubiwa** [comment]
Hey @joepio, after asking around, this is apparently a bug that was introduced in Meilisearch v0.29 and that will be fixed in Meilisearch v0.30, which is released next week:

https://github.com/meilisearch/meilisearch/pull/2861

**Comment by joepio** [comment]
@bidoubiwa  I've compiled and installed v0.30rc2, and can confirm that the issue has been fixed. Thanks!
